roxctl/common/techpreview.go

```diff
@@ -7,4 +7,4 @@ const TechPreviewLongText = `
 Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete.
 Red Hat does not recommend using them in production.
 These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
-For more information about the support scope of Red Hat Technology Preview features, see https://access.redhat.com/support/offerings/techpreview/`
+For more information about the support scope of Red Hat Technology Preview features, see https://access.redhat.com/support/offerings/techpreview/.`
```

Comment:
Should there be a whitespace between the URL and the period ? I think it would prevent strange behaviours when clicking / copying the URL.

Reply:
Nope, normally such period is not included in the URL by the parsers.